test/browser/redux/actions/async.test.js

```diff
@@ -1592,6 +1592,30 @@ describe('Asynchronous Actions', () => {
           }
         ];
         asyncActions.__Rewire__('services', {
+          api: {
+            user: {
+              profile: (cb) => {
+                cb(null, {
+                  fullName: 'Joe',
```

Comment:
These fixtures in the api mocks are not necessary (not being tested here since `clickDeviceSelectionDone` pulls the action contents out of _state_ and were actively confusing to me. I'm not sure if there are other parallel cases to look at in this codebase, but I'll be in favor of just `cb(null)` as the minimum needed in the mock.